src/components/Loader/Loader.vue

```diff
@@ -34,12 +40,62 @@ export default {
   min-width: var(--loader-size);
 }
 
-.loader.secondary {
-  --loader-size: 70px;
-  background-image: url("/img/secondary-loading.gif");
+.hide {
+  visibility: hidden;
 }
 
-.loader.hide {
-  visibility: hidden;
+.spinner-container {
+  display: flex;
+  width: 100%;
+  height: 150px;
+  justify-content: center;
+  align-items: center;
+}
+
+.loader-ripple {
+  display: inline-block;
+  position: relative;
+  width: 80px;
+  height: 80px;
+  div {
```

Comment:
Avoid using elements as selector as they might change, use classes instead

Reply:
Also, we don't support postcss-nested here (yet)

Reply:
Oh nice catch then, because in dev mode it works fine. Thanks!